include/aspect/parameters.h

```diff
@@ -312,6 +312,7 @@ namespace aspect
     double                         start_time;
     double                         CFL_number;
     double                         maximum_time_step;
+    double                         maximum_increase_time_step;
```

Comment:
I think a better name for the variable would be `relative_maximum_time_step_increase`.

Reply:
Yes, the relative should be there, but should it then be `maximum_relative_time_step_increase`?

Reply:
Sure, either is fine -- choose whatever you think is better.